Entity/FOSUser.php

```diff
@@ -18,6 +23,29 @@
      */
     protected $passwordResetToken;
 
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->enforcePasswordChange = false;
+    }
+
+    /**
+     * @return bool
+     */
+    public function hasEnforcedPasswordChange()
```

Comment:
this method name doesn't sit well with me - should be isEnforcedPasswordChange I think